src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVmss.md

```diff
@@ -27,10 +27,12 @@ You can use the *InstanceId* parameter to remove a specific virtual machine insi
 
 ### Example 1: Remove a VMSS
 ```
-PS C:\> Remove-AzureRmVmss -ResourceGroupName "Group001" -VMScaleSetName "VMScaleSet001"
+PS C:\> $vmss = Get-AzureRmVmss -ResourceGroupName $ResourceGroupName -VMScaleSetName $vmssName 
+PS C:\> Remove-AzureRmVmss -ResourceGroupName $ResourceGroupName -VMScaleSetName $vmss
```

Comment:
@cloudmelon the `$vmss` variable is set to a VMSS object, so shouldn't we be passing `$vmssName` to the `-VMScaleSetName` parameter rather than `$vmss`?